index.js

```diff
@@ -65,6 +64,24 @@ NYC.prototype._createInstrumenter = function () {
   })
 }
 
+NYC.prototype._prepExcludePatterns = function (excludes) {
+  var directories = []
+  excludes = _.map(excludes, function (exclude) {
+    // Remove leading "current folder" prefix
```

Comment:
Just did some quick testing with minimatch itself

```javascript
// minimatch(path, pattern)

minimatch('./node_modules/foo.js', 'node_modules/**') // false
minimatch('node_modules/foo.js', './node_modules/**') // false
minimatch('node_modules/foo.js', 'node_modules/**')   // true
```

Perhaps I should do as ESLint and also remove `./` from the path that is passed in?